### DISCUSSION TITLE
How do I prevent re-rendering an input when another one changes?

### DISCUSSION THREAD

**Discussion by hyphen81** [livewire/livewire#6367] [1. Help] [answered]
I have several fields on my form, some of them have `wire:model` others `wire:model.defer`.  In my app a user is doing the same repetitive data entry over and over again hundreds of times a day and is likely going to be moving through the fields using the tab key and moving very quickly. This isn't a click and wait type scenario. 

When I change one input that has `wire:model` it sends the request, which is expected. If, in the meantime, I make a change to the next input on the form while that first request is still processing, then when the first request completes, the second field is then rendered again and the change to the second input is lost. As a result, a user would have to make the change to the first field, wait for the round trip, make the change to the second, etc.

I realize I could just submit the form values after the user has made all the changes, but due to some other constraints and effects that the users selection has in the application, it's better for me to use `wire:model` if I can make it work.

Is there a way to not re-render, or some way to almost treat each field as its own separate form, independent from one another? Ideally I'd like to be able to fire the first request, then fire the second, then the third, etc, and independently update each field as each request comes back.

**Comment by N1ebieski** [comment, marked as answer]
Try the ```wire:ignore``` attribute. From the documentation:

```Instructs Livewire to not update the element or its children when updating the DOM from a server request. Useful when using third-party JavaScript libraries within Livewire components.```

**Reply by hyphen81** [reply]
well i feel stupid. I knew about that, just didn't consider it in this instance, but it worked! thanks!
